lib/tasks/db.rake

```diff
@@ -3,15 +3,16 @@
 namespace :db do
   task recreate: ['invoker:stop_all', 'rabbitmq:purge', 'repo:clean'] do
     Rake::Task['db:recreate'].invoke
+    Rake::Task['index:clear'].invoke
```

Comment:
I think instead of defining `index:clear`, you can just use [`chewy:reset`](https://github.com/toptal/chewy#rake-tasks).